Weather/README.md

```diff
@@ -1,10 +1,10 @@
 # Weather data models
 
-This folder contains all the code that supports NGSIv2 end points for providing weather information.
+This folder contains all the code that defines domain-specific datamodels for providing weather information.
 The following entity types are supported:
 
 * `WeatherObserved` .- Weather observations provided by [AEMET Weather Stations](../PointOfInterest/WeatherStation/README.md)
 * `WeatherForecast` .- Weather forecasts provided by [AEMET](http://aemet.es) and [IPMA](http://ipma.pt)
-* `WeatherAlarm` .- Weather alarms provided by [MeteoAlarm](http://meteoalarm.eu) and [IPMA](http://ipma.pt)
+* `Alert` .- Weather alarms provided by [MeteoAlarm](http://meteoalarm.eu) and [IPMA](http://ipma.pt)
```

Comment:
->
* `Alert` .- Weather alarms provided by [MeteoAlarm](http://meteoalarm.eu) and [IPMA](http://ipma.pt). Weather alarms are modelled using the generic [Alert](../../Alert/doc/spec.md) model.